rts/Rendering/Textures/Bitmap.cpp

```diff
@@ -1619,7 +1619,7 @@ static void HandleDDSMipmap(GLenum target, bool mipmaps, int num_mipmaps)
 	}
 }
 
-unsigned int CBitmap::CreateDDSTexture(unsigned int texID, float aniso, float lodBias, bool mipmaps) const
+unsigned int CBitmap::CreateDDSTexture(unsigned int texID, bool mipnear, float aniso, float lodBias, bool mipmaps) const
```

Comment:
I think it's time to move CBitmap `Create*` functions to use the parameters structure.